next.config.js

```diff
@@ -12,4 +12,23 @@ const withNextra = require('nextra')({
   }
 })
 
-module.exports = withNextra()
+module.exports = withNextra(
```

Comment:
Esta forma de implementar el código es incorrecta lo que puede causar errores en la pagina web te recomiendo la siguiente forma de implementar el codigo:
```suggestion
module.exports = withNextra({
   redirects: function () {
      return [
         // Redirecciones
      ]
   }
})
```